docs/tasks/archive/2026/07/20260731-board-whiteboard-elements-todo.md

```
@@ -1310,33 +1310,33 @@ git commit -m "Add board minimap overlay with viewport rect + drag-to-pan"
 
-- [ ] **Step 3: Manual smoke in `pnpm dev`**
+- [x] **Step 3: Manual smoke in `pnpm dev`**
 
 Start `docker compose up -d` (if not running) + `pnpm dev`. Open/create a board (`/b/:id`) and verify:
   - Sticky: click the Sticky button → a yellow 180² sticky appears at screen center, selected, in text-edit; type text → it stays centered + shrinks to fit. Chevron palette drops other colors. Move/resize/rotate work (it's a shape). Reload → sticky + text persist (CRDT).
   - Image: drag a PNG onto the canvas → it uploads and lands at the viewport center. Paste an image from the clipboard → same. Toolbar Image button → file picker → inserts. Pasting *inside* a sticky's text does NOT drop a canvas image.
   - Minimap: bottom-right overview shows all elements; the blue rectangle tracks pan/zoom; dragging in the minimap recenters the board; toggle hides/shows it. Empty board → minimap panel is blank (no crash).
   - Collab (optional): open the same board in a second tab → a sticky/image added in one appears in the other.
 
-- [ ] **Step 4: Self code-review over the branch diff**
+- [x] **Step 4: Self code-review over the branch diff**
```

markdownlint-cli2 (0.23.2)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)